dashboard/app/views/levels/show.html.haml

```diff
@@ -25,7 +25,7 @@
       %h4{style: 'margin-left: auto; margin-right: auto; max-width: 800px;'}!= @level.properties['instructions']
   -# Pass special parameters to Blockly partial for embedded level type.
   = render partial: 'levels/blockly'
-- elsif @level.is_a?(DSLDefined) || @level.is_a?(Widget)
+- elsif @level.is_a?(DSLDefined) || @level.is_a?(Widget) || @level.is_a?(OpenResponse)
```

Comment:
This whole section will need a refactor at some point.  I think it's ok to add this case for now.  Eventually `initApp.js` should know about non-Blockly level types and be able to handle them appropriately.